lib/geocoding.rb

```diff
@@ -8,15 +8,24 @@ def initialize(location)
   end
 
   def coordinates
+    geocoded_location = begin
+      Geocoder.coordinates(location, components: "country:uk")
```

Comment:
@josephhull676's original PR changed this to `components: "country:GB"` - I think GB is the correct country code.

Reply:
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#GB

Reply:
We're so special we have 2, but I can go with `gb` everywhere instead 👍